recipe/meta.yaml

```
@@ -1,43 +1,47 @@
-{% set version = "1.4.3" %}
+{% set name = "rasterio" %}
+{% set version = "1.4.4" %}
 
 package:
-  name: rasterio
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/mapbox/rasterio/archive/{{ version }}.tar.gz
-  sha256: dfa248fdc46f0e08ab957712b13a86d7cd49bce72f34b7fdca94af3b2d850181
+  url: https://github.com/mapbox/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 056bc5ad56600f27fbebdaec76ec8c1ff66f2adf6b54d33fabb816d1a9e914d5
 
 build:
   number: 0
-  skip: True  # [py<38 or s390x]
+  skip: True  # [py<310]
   entry_points:
     - rio = rasterio.rio.main:main_group
 
 requirements:
   build:
+    - {{ stdlib("c") }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - cython
```

```suggestion
    - cython 3.1.*
```

Ref: https://github.com/rasterio/rasterio/blob/1.4.4/pyproject.toml#L5